### PR TITLE
Adjust HEMFluidProperties for use in Bison.

### PR DESCRIPTION
**Pull request by jasondhales**
A matching merge request is in for IAPWS95.
References #10549

### CODE REVIEW

File: modules/fluid_properties/include/userobjects/HEMFluidProperties.h [hunk 1]
```diff
@@ -56,6 +56,7 @@ class HEMFluidProperties : public FluidProperties
    * @param e Specific internal energy
    */
   virtual Real quality(Real v, Real e) const = 0;
+  virtual Real quality_Tsat_h(Real Tsat, Real h) const = 0;
```
Comment (andrsd) [inline modules/fluid_properties/include/userobjects/HEMFluidProperties.h]: Add doco for this method.

File: modules/fluid_properties/include/userobjects/HEMFluidProperties.h [hunk 2]
```diff
@@ -122,7 +123,7 @@ class HEMFluidProperties : public FluidProperties
    * @param pressure pressure
    * @param temperature temperature
    */
-  virtual Real rho(Real pressure, Real temperature) const = 0;
+  virtual Real rho(Real pressure, Real temperature, Real quality) const = 0;
```
Comment (andrsd) [inline modules/fluid_properties/include/userobjects/HEMFluidProperties.h]: Add `@param quality ...` doco.

File: modules/fluid_properties/include/userobjects/HEMFluidProperties.h [hunk 3]
```diff
@@ -169,7 +170,7 @@ class HEMFluidProperties : public FluidProperties
    * @param pressure pressure
    * @param temperature temperature
    */
-  virtual Real h(Real pressure, Real temperature) const = 0;
+  virtual Real h(Real pressure, Real temperature, Real quality) const = 0;
```
Comment (andrsd) [inline modules/fluid_properties/include/userobjects/HEMFluidProperties.h]: Add `@param quality ...` doco.

**Review by andrsd — CHANGES_REQUESTED** [review]

**Comment by jasondhales** [comment]
I added the documentation.

**Review by andrsd — APPROVED** [review]

**Pull request merged**